CoastalResilienceNetwork/GeositeFramework
# Remove layer_selector plugin

#### RIP Original layer selector.  
This plugin has been removed as a core plugin, but is still available at
https://github.com/CoastalResilienceNetwork/map-tree and can be included
through a build process.  Additional necessary changes:
* Remove references is core region framework to layer_selctor
* Remove sample_plugin which showed how to extend layer_selector
* Move library `tv4`, which was included with layer_selector, but was
  refereced via `use!` by multiple other plugins, to the core framework.

Connects https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/653

#### Testing
* Verify the site loads and the tests pass
* Verify layer_selector is not found
* Add map tree into a `layer_selector` directory in `plugins/`, 
```
git clone git@github.com:CoastalResilienceNetwork/map-tree.git layer_selector
``` 
  and verify it still works
